package/MDAnalysis/analysis/align.py

```diff
@@ -663,9 +667,7 @@ def __init__(self, mobile, reference, select='all', filename=None,
 
         # do this after setting the memory reader to have a reference to the
         # right reader.
-        super(AlignTraj, self).__init__(mobile.trajectory, **kwargs)
-        if not self._verbose:
-            logging.disable(logging.WARN)
```

Comment:
I had to move this to the `_prepare` method. Will this cause any issues?

Reply:
Yes, because `get_matching_atoms()` uses logging and the output is important for debugging.

This is actually one of the cases that @richardjgowers talked about when he said "How would we know what happens in init?" (and where @IAlibay and myself said "Bah, humbug, nothing ever happens in init."

(Sorry for the *slight* misquote @IAlibay )

This is also an example where it makes sense that init does a stuff that you wouldn't necessarily want to do in `_prepare` every time. Nevertheless, it could be rewritten so that `_prepare` only does things once.